setup.py

```diff
@@ -65,14 +69,14 @@
 
 def include_documentation(local_dir, install_dir):
     global DATA_FILES
-    if "bdist_wheel" in sys.argv and not path.exists(local_dir):
-        print(
-            "Directory '{}' does not exist. "
-            "Please build documentation before running bdist_wheel.".format(
-                path.abspath(local_dir)
-            )
-        )
-        sys.exit(0)
+    # if "bdist_wheel" in sys.argv and not path.exists(local_dir):
```

Comment:
@JakaKokosar were there problems with tests that you commented this out? What exactly is the reason for it?

Reply:
Yeah, it breaks building package when running tox. This is just a temporary workaround. When we finish migrating from Travis to GH actions/tox we will make sure that building docs will be a part of a package build process (similar to what Aleš has done with Orange).

Reply:
Ok, great.